<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Convert">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>160</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FullView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>590</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>FULL View</string>
    </property>
   </widget>
   <widget class="QLabel" name="original">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Preview">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>471</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>471</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Filtered</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>901</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Filters</string>
    </property>
    <widget class="QRadioButton" name="V_gradient_3">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>61</y>
       <width>138</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Verical gradient</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Low_3">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>33</y>
       <width>94</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Low Path</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Sharpen">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>89</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Sharpen</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="H_gradient_3">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>61</y>
       <width>164</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal gradient</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="High_3">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>33</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>High Path</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Laplacien">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>89</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Laplacien</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="v_prewitt">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertical Prewitt</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="H_prewitt">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal Prewitt</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="V_sobel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Verical Sobel</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="H_sobel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal Sobel</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Default">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Flou">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Flou</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Rehausseur">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rehausseur</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="FeiChen">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Frei-Chen</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/new/prefix1/Images/Open-icon.png</normaloff>:/new/prefix1/Images/Open-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/new/prefix1/Images/Save-icon.png</normaloff>:/new/prefix1/Images/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/new/prefix1/Images/exit-icon.png</normaloff>:/new/prefix1/Images/exit-icon.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/new/prefix1/Images/help-icon.png</normaloff>:/new/prefix1/Images/help-icon.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/new/prefix1/Images/about.png</normaloff>:/new/prefix1/Images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
